client/tests/integration/editbird.test.js

```diff
@@ -0,0 +1,16 @@
+describe.skip('Editbird', () => {
+  beforeAll(async () => {
+    await page.goto('http://localhost:8000/');
```

Comment:
It could be useful to have wagtail's base url in an environment variable. 
I often expose wagtail's port on another port than 8000 in order to try things.

I am not sure if it is relevant here though

Reply:
Good point, definitely!

Reply:
I’ve added a `TEST_ORIGIN=http://localhost:9000` environment variable so we can both test on different ports when running locally, and test remote Wagtail instances.